connector/spanmetricsconnector/metadata.yaml

```
@@ -6,7 +6,7 @@ status:
     alpha: [traces_to_metrics]
   distributions: [contrib, sumo, grafana, splunk, liatrio]
   codeowners:
-    active: [albertteoh, portertech]
+    active: [portertech]
```

```suggestion
    active: [portertech]
    emeritus: [albertteoh]
```
`make generate` may need to be run after this change as well.